backend/store/etcd/kvc/client.go

```diff
@@ -0,0 +1,54 @@
+package kvc
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/sensu/sensu-go/backend/store"
+	"github.com/sensu/sensu-go/util/retry"
+)
+
+const (
+	// EtcdRoot is the root of all sensu storage.
+	EtcdRoot = "/sensu.io"
+
+	// EtcdInitialDelay is 100 ms.
+	EtcdInitialDelay = time.Millisecond * 100
+
+	// NamespacesPathPrefix is the path to namespaces in etcds
+	NamespacesPathPrefix = "namespaces"
```

Comment:
For `NamespacePathPrefix` and `EtcdRoot`, shouldn't those default come from the package that define them (`backend/store/etcd`, though the former would have to be exported)? That way, they can never be inconsistent.

Reply:
The "problem" is that both the v1 & v2 etcd stores need those, and if we moved them into one of these packages, we would probably have an import cycle hell. WDYT?